### PR TITLE
Export syncTimeout in parts/Utilities.ts

### PR DESCRIPTION
**Pull request by jon-a-nygaard**
Title says it all.

Please take an extra look at the function overloads to make sure these are good enough.

### CODE REVIEW

**Comment by bre1470** [comment]
I agree. This can be reduced to a single function.

**Comment by jon-a-nygaard** [comment]
@TorsteinHonsi and @bre1470 please see latest changes.

File: ts/parts/Utilities.ts [hunk 1]
```diff
@@ -1525,26 +1525,27 @@ function splat(obj: any): Array<any> {
...
  */
-H.syncTimeout = function (
+function syncTimeout(
     fn: Function,
-    delay?: number,
-    context?: any
-): (number|undefined) {
-    if (delay) {
+    delay: number,
+    context?: unknown
```
Comment (bre1470) [inline ts/parts/Utilities.ts]: Type in namespace has to change as well.

Comment (jon-a-nygaard) [reply ts/parts/Utilities.ts]: Thanks, should be fixed in last update.

**Comment by jon-a-nygaard** [comment]
I see there is now a conflict between master and this branch, I will solve it locally and force push an updated version of the branch.

**Pull request merged**